src/go/k8s/helm-chart/charts/redpanda-operator/values.yaml

```diff
@@ -0,0 +1,81 @@
+# Default values for redpanda-operator.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# nameOverride -- Override name of app
+nameOverride: ""
+
+# fullnameOverride -- Override the fully qualified app name
+fullnameOverride: ""
+
+# replicaCount -- Number of instances of Redpanda Operator
+replicaCount: 1
+
+image:
+  # image.repository -- Repository that Redpanda Operator image is available
+  repository: vectorized/redpanda-operator
+  # image.tag -- Define the Redpanda Operator container tag
+  tag: latest
```

Comment:
Sorry I missed this on previous reviews - I want to clarify the versioning also.

What justifies helm chart version bump?
If we have a certain app version of this, why do we point to latest here? We should point to that version, right?

Reply:
You are right. I will change that to the git tag.